tapir/coop/templates/coop/welcome_desk_member.html

```diff
@@ -0,0 +1,57 @@
+{% extends "accounts/base.html" %}
+
+{% load bootstrap4 %}
+{% load static %}
+{% load i18n %}
+{% load shifts %}
+{% load coop %}
+{% load log %}
+{% load accounts %}
+
+{% block head %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static "shifts/css/shifts.css" %}">
+    <link rel="stylesheet" href="{% static "accounts/css/user_detail.css" %}">
+{% endblock %}
+
+{% block content %}
+    <div class="card m-2" id="welcome_desk_member">
+        <h5 class="card-header">
+            <span>{{ member.get_info.get_display_name }}</span>
+        </h5>
+        <div class="card-body">
+            {% if can_shop %}
+                <div class="alert alert-success" role="alert" id="welcome_desk_can_shop">
+                    {% blocktranslate with name=member.get_info.first_name %}{{ name }}  can shop :){% endblocktranslate %}
+                </div>
+            {% else %}
+                {% if missing_tapir_account %}
+                    <div class="alert alert-danger" role="alert" id="welcome_desk_no_tapir_account">
+                        {% blocktranslate with name=member.get_info.first_name %}{{ name }} does not have a Tapir account. Contact a member of the management team.{% endblocktranslate %}
```

Comment:
Maybe add a bit more info here about what this means? Something like "This means that they can not actively participate in the shift system."